### PR TITLE
[#336] Add scripts to deploy the operator

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

File: deploy/cluster_wide_install_opr.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+echo "Deploying cluster-wide operator, dont forget change WATCH_NAMESPACE to empty! and cluser-role-binding subjects namespace"
+
+read -p "Please confirm that you have changed WATCH_NAMESPACE and role-binding to proper values [Y/N]: "
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+  exit 1
+fi
+
+KUBE=oc
```
Comment (gaohoward) [inline deploy/cluster_wide_install_opr.sh:12]: it probably default to 'kubectl' ?

Comment (pwright) [reply deploy/cluster_wide_install_opr.sh:12]: +1 to kubectl

**Comment by brusdev** [comment]
> will this actually work on minikube. I switched the KUBE to kubectl and ran install_opr.sh but:
> 
> ```
> $ kubectl get pods
> NAME                                                  READY   STATUS             RESTARTS   AGE
> activemq-artemis-controller-manager-db6db4f59-lrjjx   0/1     ImagePullBackOff   0          2m42s
> ```

this is because we haven' t release yet quay.io/artemiscloud/activemq-artemis-operator:1.0.6
you could change the image defined in deploy/operator.yaml to `quay.io/artemiscloud/activemq-artemis-operator:1.0.5`

File: deploy/install_opr.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Deploying operator to watch single namespace"
+
+KUBE=oc
```
[the remark this answers is not in the document]
Comment (brusdev) [reply deploy/install_opr.sh]: repleced

File: deploy/cluster_wide_install_opr.sh (hunk 1 above)
Comment (brusdev) [reply deploy/cluster_wide_install_opr.sh:12]: fixed

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -37,21 +37,22 @@ namespace.
...
 ## Deploying the operator
 
-To deploy the operator simply run
+To deploy the operator in the current namespace simply run
 
-```$xslt
-$ make deploy
+```shell
+cd deploy
+./install_opr.sh
```
Comment (pwright) [inline docs/getting-started/quick-start.md:44]: how do you check whether this worked?

I misread the current text and tried ` kubectl get pod -n activemq-artemis-operator` but that's only applicable if you're doing a cluster wide deploy

**Pull request merged**